ote/src/clj/ote/services/transport.clj

```diff
@@ -87,25 +80,37 @@
                 add-error-data))
           modified-services)))
 
+(defn- append-ote-netex-interfaces [{::t-service/keys [id] :as service} config db]
+  (assoc service
+    :ote-interfaces
+    (vec
+      (sort-by :url
+               (mapv #(assoc %
+                        :url (export-netex/file-download-url config id (:ote.db.netex/id %))
```

Comment:
:ote.db.netex could be changed to ::netex and import ote.db.netex.

Reply:
Reason for not doing that was a namespace coflict with integration.export.netex